### PR TITLE
Fixed: Wrong namespace in advanced app backend unit test classes #6919

### PR DESCRIPTION
**Pull request by ivokund**
Fixed namespace declaration in two classes and added a README entry

### CODE REVIEW

File: framework/CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Yii Framework 2 Change Log
 - Enh #6697: Added `yii\helpers\Url::current()` method that allows adding or removing parameters from current URL (samdark, callmez)
 - Chg #5690: adjusted paths in message config generated by `yii message/config` to reflect directory structure better (mikehaertl, samdark)
 - Chg #6661: Hyperlinks that are enclosed within an exist form will use the same form for submission if they specify both of the `href` and `data-method` attributes (qiangxue)
-
+- Bug #6919: Fixed wrong namespaces under advanced application's TestCase classes (ivokund)
```
Comment (Ragazzo) [inline framework/CHANGELOG.md]: `bug` section should go as first

**Comment by Ragazzo** [comment]
Can be merged , valid fix

File: framework/CHANGELOG.md (hunk 1 above)
Comment (ivokund) [inline framework/CHANGELOG.md]: Changed the order

**Comment by samdark** [comment]
Thanks!

**Pull request merged**